docs/cloud/getting-started-with-airbyte-cloud.md

```diff
@@ -83,6 +89,12 @@ If you need to use [cron scheduling](http://www.quartz-scheduler.org/documentati
 
 To set up a connection:
 
+:::note
+
+Set your [default data residency](https://docs.airbyte.com/cloud/managing-airbyte-cloud/manage-data-residency#choose-your-default-data-residency) before creating a new connection to ensure your data is processed in the correct region.
+
```

Comment:
@Amruta-Ranade Is it overkill to have this note twice in the Getting Started doc? I added it to both the "To set up a source" and "To set up a connection" sections because it applies to both, but if the users are following the instructions they would have probably set their default data residency before they set up their source.

Reply:
We do need the note in both places. Users won't always read the instructions in order. What if they already had the source set up and are reading this guide to figure out how to set up a connection?